### ISSUE TITLE
mouseUpHandler in a custom control is rarely triggered if Apple Pencil is used

### ISSUE THREAD

**Issue by ct-gilford** [fabricjs/fabric.js#8042]
### Discussed in https://github.com/fabricjs/fabric.js/discussions/8041

<div type='discussions-op-text'>

<sup>Originally posted by **ct-gilford** July  1, 2022</sup>
I created a custom polygon class that is a triangle that can show/hide angles by clicking on the corresponding vertex.

To toggle show/hide angle, I created a custom control that is placed on each vertex of the triangle. Clicking the custom control will trigger **mouseUpHandler** which will then toggle showing/hiding the angle. This work fine on **desktop with mouse click** as well as on **ipad with touch**. But if I use **Apple Pencil** to tap on the custom control, it will trigger **actionHandler** more often than **mouseUpHandler**. **actionHandler** is also implemented so that user can drag the vertex to change the shape of the triangle.

In other words, whenever **Apple Pencil** is used to tap on the custom control, even with a quick tap, it will almost always drag the vertex to change the shape of the triangle instead of showing/hiding the angle. This doesn't happen with mouse click on desktop and touch on ipad.

Is there a way to make **Apple Pencil** to work with **mouseUpHandler** as well? If not, what is the alternative that you can suggest for me to achieve the same effect.</div>
